app/driver/onewire.c

```diff
@@ -180,7 +180,6 @@ void onewire_write(uint8_t pin, uint8_t v, uint8_t power /* = 0 */) {
   if ( !power) {
```

Comment:
Up to you.  I will be doing a pass later to remove all of these redundant interrupt locks some  time in the future.